tests/test_runner/test_discover_runner.py

```diff
@@ -269,14 +269,6 @@ def test_pdb_with_parallel(self):
         with self.assertRaisesMessage(ValueError, msg):
             DiscoverRunner(pdb=True, parallel=2)
 
-    def test_buffer_with_parallel(self):
-        msg = (
-            'You cannot use -b/--buffer with parallel tests; pass '
-            '--parallel=1 to use it.'
-        )
-        with self.assertRaisesMessage(ValueError, msg):
-            DiscoverRunner(buffer=True, parallel=2)
-
```

Comment:
Can we add a test? similar to the `test_buffer_mode_test_pass()`.

Reply:
I tried:

```python
    def test_buffer_mode_parallel(self):
        runner = DiscoverRunner(parallel=2, buffer=True, verbose=0)
        with captured_stdout() as stdout, captured_stderr() as stderr:
            suite = runner.build_suite([
                'test_runner_apps.buffer.tests_buffer.WriteToStdoutStderrTestCase.test_pass',
                'test_runner_apps.buffer.tests_buffer.WriteToStdoutStderrTestCase2.test_pass',
            ])
            runner.run_suite(suite)
        self.assertNotIn('Write to stderr.', stderr.getvalue())
        self.assertNotIn('Write to stdout.', stdout.getvalue())
```

But this doesn't work because we can't capture stdin/stdout across the parallel processes. Any suggestions?